cmd/receptor.go

```diff
@@ -83,11 +83,16 @@ func (cfg nullBackendCfg) Run() error {
 }
 
 func main() {
-	cmdline.AddConfigType("node", "Node configuration of this instance", nodeCfg{}, cmdline.Required, cmdline.Singleton)
-	cmdline.AddConfigType("local-only", "Run a self-contained node with no backends", nullBackendCfg{}, cmdline.Singleton)
-	cmdline.AddConfigType("version", "Show the Receptor version", version.CmdlineCfg{}, cmdline.Exclusive)
-	whatRan := cmdline.ParseAndRun(os.Args[1:], []string{"Init", "Prepare", "Run"})
-	if whatRan != "" {
+	cl := cmdline.GlobalInstance()
+	cl.AddConfigType("node", "Node configuration of this instance", nodeCfg{}, cmdline.Required, cmdline.Singleton)
+	cl.AddConfigType("local-only", "Run a self-contained node with no backends", nullBackendCfg{}, cmdline.Singleton)
+	cl.AddConfigType("version", "Show the Receptor version", version.CmdlineCfg{}, cmdline.Exclusive)
+	err := cl.ParseAndRun(os.Args[1:], []string{"Init", "Prepare", "Run"}, cmdline.ShowHelpIfNoArgs)
+	if err != nil {
+		fmt.Printf("Error: %s\n", err)
```

Comment:
I'm trying not to break existing behavior too much, but this should arguably be:
```
_, _ = fmt.Fprintf(os.Stderr, "Error: %s\n", err)
```
so that errors go to stderr.